style.css

```diff
@@ -1125,3 +1090,100 @@ body {
   background-size: 30px 30px;
   background-attachment: local;
 }
+
+/* **********form****** */
+.footer {
+  display: flex;
+  grid-template-columns: 1fr;
+  position: inherit;
+  width: 350px;
+  top: 2200px;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 200px;
+  height: 30px;
+  flex: none;
+  list-style: none;
+  margin-left: -10px;
+}
+
+.footer ul li {
+  width: 60px;
+}
+
+.form {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-flow: minmax(150px, auto);
+  position: inherit;
+  width: 375px;
+  height: auto;
+  top: 2100px;
+  background-color: #fff;
+}
```

Comment:
Please ensure flexbox is used to layout the `form element` according to the project requirement.

Reply:
@micheaol Thank you for reviewing my project, I will make the necessary corrections as suggested